README.md

```diff
@@ -20,15 +20,30 @@ This helm chart installation will create a single-node MarkLogic cluster with a
 ```
 helm repo add marklogic https://marklogic.github.io/marklogic-kubernetes/
 ```
-2. Set below parameters in the values.yaml to allocate compute resources
+
+2. Create a secret to store MarkLogic admin credentials that includes username, password and wallet-password:
+Substitute your choice of values for admin credentials in the below command:
+```
+kubectl create secret generic ml-admin-secrets \
+    --from-literal=adminUsername='' \
+    --from-literal=adminPassword='' \
+    --from-literal=walletPassword=''
+```
```

Comment:
Maybe we should specify the namespace to create the secret.
-n <release-namespace>